run.py

```
@@ -0,0 +1,42 @@
+
+# Import the ASGI application after Django is set up
+from web.asgi import application
+
+def signal_handler(sig, frame):
+    print("\nShutting down server...")
+    sys.exit(0)
+
+if __name__ == "__main__":
+    # Register signal handler
+    signal.signal(signal.SIGINT, signal_handler)
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)